data.sql

```diff
@@ -11,3 +11,40 @@ INSERT INTO ANIMALS (id,name, date_of_birth,neutered,escape_attempts,weight_kg)
 INSERT INTO ANIMALS (id,name, date_of_birth,neutered,escape_attempts,weight_kg) VALUES (9,'Boarmon','2005-06-07',true,7,20.4);
 INSERT INTO ANIMALS (id,name, date_of_birth,neutered,escape_attempts,weight_kg) VALUES (10,'Blossom','1998-10-13',true,3,17);
 INSERT INTO ANIMALS (id,name, date_of_birth,neutered,escape_attempts,weight_kg) VALUES (11,'Ditto','2022-05-14',true,4,22);
+
+INSERT INTO owners values(default, 'Jennifer Orwell', 19);
+INSERT INTO owners values(default, 'Bob', 45);
+INSERT INTO owners values(default, 'Melody Pond', 77);
+INSERT INTO owners values(default, 'Dean Winchester', 14);
+INSERT INTO owners values(default, 'Jodie Whittaker', 38);
+
+INSERT INTO species (name) values ('Pokemon'),('Digimon');
+update animals set species_id=2 where name like '%mon';
+update animals set species_id=1 where species_id is null;
+
+UPDATE animals SET owner_id=1 WHERE name='Agumon';
+UPDATE animals SET owner_id=2 WHERE name='Gabumon' OR name = 'Pikachu';
+UPDATE animals SET owner_id=3 WHERE name='Devimon' or name = 'Plantmon';
+UPDATE animals SET owner_id=4 WHERE name='Charmander' OR name = 'Squirtle' OR name = 'Blossom';
+UPDATE animals SET owner_id=5 WHERE name='Angemon' OR name = 'Boarmon';
+
+INSERT INTO visits (animals_id, vets_id, date_of_visit) VALUES (1, 1, '2020-05-24'),
+INSERT INTO visits (animals_id, vets_id, date_of_visit) VALUES (1, 3, '2020-07-22'),
+INSERT INTO visits (animals_id, vets_id, date_of_visit) VALUES (2, 4, '2021-02-02'),
+INSERT INTO visits (animals_id, vets_id, date_of_visit) VALUES (3, 2, '2020-01-05'),
+INSERT INTO visits (animals_id, vets_id, date_of_visit) VALUES (3, 2, '2020-03-08'),
+INSERT INTO visits (animals_id, vets_id, date_of_visit) VALUES (3, 2, '2020-05-14'),
+INSERT INTO visits (animals_id, vets_id, date_of_visit) VALUES (4, 3, '2021-05-04'),
+INSERT INTO visits (animals_id, vets_id, date_of_visit) VALUES (5, 4, '2021-02-24'),
+INSERT INTO visits (animals_id, vets_id, date_of_visit) VALUES (6, 2, '2019-12-21'),
+INSERT INTO visits (animals_id, vets_id, date_of_visit) VALUES (6, 1, '2020-08-10'),
+INSERT INTO visits (animals_id, vets_id, date_of_visit) VALUES (6, 2, '2021-04-07'),
+INSERT INTO visits (animals_id, vets_id, date_of_visit) VALUES (7, 3, '2019-09-29'),
+INSERT INTO visits (animals_id, vets_id, date_of_visit) VALUES (8, 4, '2020-10-03'),
+INSERT INTO visits (animals_id, vets_id, date_of_visit) VALUES (8, 4, '2020-11-04'),
+INSERT INTO visits (animals_id, vets_id, date_of_visit) VALUES (9, 2, '2019-01-24'),
+INSERT INTO visits (animals_id, vets_id, date_of_visit) VALUES (9, 2, '2019-05-15'),
+INSERT INTO visits (animals_id, vets_id, date_of_visit) VALUES (9, 2, '2020-02-27'),
+INSERT INTO visits (animals_id, vets_id, date_of_visit) VALUES (9, 2, '2020-08-03'),
+INSERT INTO visits (animals_id, vets_id, date_of_visit) VALUES (10, 3, '2020-05-24'),
+INSERT INTO visits (animals_id, vets_id, date_of_visit) VALUES (10, 1, '2021-01-11');
```

Comment:
- [ ] I can see you put in a lot of work into this project. I noticed that you didn't insert data into the `vets` table and the `specializations` table. You need to insert the correct data into these tables so that your queries give the correct results. Please implement this change. 👍 😎